### PR TITLE
Add meson glib post install

### PR DESCRIPTION
**Pull request by worldofpeace**

### CODE REVIEW

File: meson/post_install.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python3
+
+import os
+import subprocess
+
+schemadir = os.path.join(os.environ['MESON_INSTALL_PREFIX'], 'share', 'glib-2.0', 'schemas')
+
+if not os.environ.get('DESTDIR'):
+    print('Compiling gsettings schemas...')
+subprocess.call(['glib-compile-schemas', schemadir])
```
[the remark this answers is not in the document]
Comment (worldofpeace) [reply meson/post_install.py]: yep

**Comment by worldofpeace** [comment]
Did the thing

**Comment by danrabbit** [comment]
Thanks for the fix!

**Comment by worldofpeace** [comment]
I also believe [photos](https://github.com/elementary/photos) needs this also?

**Pull request merged**